docs/data/introduction/licensing/licensing.md

```diff
@@ -219,6 +219,15 @@ export default function MyApp(props) {
 }
 ```
 
+#### Install the key as an environment variable in Next.js
```

Comment:
As I understand it, the values in the .env shouldn't be in git, developers on the team have to find the value, somehow.

<img width="658" alt="Screenshot 2023-07-11 at 00 04 21" src="https://github.com/mui/mui-x/assets/3165635/a725d368-67da-4f63-91d1-4bc0493d93a1">

https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#default-environment-variables

I have mixed feelings about it:

- On one end, this feels extra complicated, do you really want to give each of your developers the license key for their env? It's a pain to set up, it's a pain to change the key after each renewal.
- On the other end, it has the potential to allow keeping track of the use, you can count each time a developer asks for it.

I suspect that >90% of the time when developers use the env variable, they commit it to git, pointless. So maybe we should warn that we don't recommend it, unless for use cases where the key is not in git:

1. Source available, need to keep the license key private to their org.
2. The desire to have a tighter control on who in the organization can access the license key

Reply:
Alright, I didn't intend to recommend using environment variables.
Maybe we can change the section title:

```suggestion
#### If you need to install the key as an environment variable in Next.js
```